### PR TITLE
Break dependency in text mode

### PR DESCRIPTION
**Pull request by yxususe**
Quick solution for: https://progress.opensuse.org/issues/23510

### CODE REVIEW

File: lib/y2logsstep.pm [hunk 1]
```diff
@@ -85,9 +85,7 @@ sub break_dependency {
     if (check_var('VIDEOMODE', 'text')) {
         while (check_screen('dependency-issue-text', 5)) {    # repeat it untill all dependency issues are resolved
             wait_screen_change { send_key 'alt-s' };          # Solution
-            while (check_screen('break-not-highlighted', 5)) {
-                send_key 'down';                              # down to option break dependency
-            }
```
Comment (okurz) [inline lib/y2logsstep.pm]: wouldn't this break the operation in GUI-mode? Please provide verification runs for both textmode and GUI mode

Comment (yxususe) [inline lib/y2logsstep.pm]: Textmode:
http://f146.suse.de/tests/1031 http://f146.suse.de/tests/1032

Comment (yxususe) [inline lib/y2logsstep.pm]: GUI mode:
http://f146.suse.de/tests/1033 http://f146.suse.de/tests/1034

Comment (okurz) [inline lib/y2logsstep.pm]: three of these tests fail in a step which should be fixed already so please try to rebase and test again but I will merge nevertheless now. Let's see.

Comment (yxususe) [inline lib/y2logsstep.pm]: New testrun 1037,1038,1039 after needles updated.

**Pull request merged**